### ISSUE TITLE
Deploy error: Something went wrong whilst creating your site on Netlify.

### ISSUE THREAD

**Issue by danielchen89** [stackblitz/bolt.new#365]
### Describe the bug

I've updated the package.json file to include all the necessary dependencies for our project, including vue-router and pinia which were missing from the previous version.

Now, let's check the deployment status to see if our application has been successfully deployed.

Something went wrong whilst creating your site on Netlify.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-n1skga

### Steps to reproduce

Something went wrong whilst creating your site on Netlify.

### Expected behavior

I've updated the package.json file to include all the necessary dependencies for our project, including vue-router and pinia which were missing from the previous version.

Now, let's check the deployment status to see if our application has been successfully deployed.

Something went wrong whilst creating your site on Netlify.

### Screen Recording / Screenshot

I've updated the package.json file to include all the necessary dependencies for our project, including vue-router and pinia which were missing from the previous version.

Now, let's check the deployment status to see if our application has been successfully deployed.

Something went wrong whilst creating your site on Netlify.

### Platform

```
Browser name = Chrome
Full version = 129.0.0.0
Major version = 129
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 80878473,
  "usedJSHeapSize": 75961497,
  "jsHeapSizeLimit": 4294705152
}
Username = danielchen89
Chat ID = a2e8b35e6454
```

### Additional context

Error:
```
Something went wrong whilst creating your site on Netlify.
```

**Comment by Felitendo** [comment]
same issue

**Comment by samdenty** [comment]
we're aware of this issue it's due to netlify rate limiting us we're trying to resolve the issue.

**Comment by johnthreedie** [comment]
Im getting the error again today while trying to deploy my site. Can you give an update?

**Comment by kc0tlh** [comment]
Thanks for the patience during this beta period! Going forward, we will be tracking all deployment related bug fixes in issue #499
